### DISCUSSION TITLE
When 'Public document sharing' of a collection is disabled, allow user to share doc link to team members in an intuitive way

### DISCUSSION THREAD

**Discussion by victorhuberta** [outline/outline#2443] [Ideas]
The sharing feature is very intuitive and easy to use when 'Public document sharing' for a collection is enabled. But when it's disabled, the 'Share' button disappears and the user has to manually copy the browser URL and send it to their teammate. Afterwards, the teammate opens the URL and is greeted with the same page. The problem is that unlike with the usual share link, this page displays the document with the sidebar and looks busy.

I think it would be great if the 'Share' button doesn't disappear when 'Public document sharing' is disabled, and the user can share the link in precisely the same way (but they cannot enable public view, so only for teammates). Then when their teammates click on the link, they will be greeted by the document in a clean manner, without the sidebar. What do you guys think?

**Comment by tommoor** [comment]
Agreed, I created an issue from this

**Reply by victorhuberta** [reply]
Thanks!
